app/interfaces/api/router/contract.go

```diff
@@ -91,6 +108,7 @@ func sendContract(contractsService *apiservices.ContractService) func(w http.Res
 func NewContractsRouter(contractsService *apiservices.ContractService) http.Handler {
 	r := chi.NewRouter()
 	r.Get("/", getAllStoredContracts(contractsService))
+	r.Get("/{contractID}", getContractByID(contractsService))
```

Comment:
I think we decided to use `contractId` instead of `contractID`. You might just as well call it `id` for simplicity